CefSharp.props

```diff
@@ -5,10 +5,11 @@
     <VisualStudioProductVersion>2013</VisualStudioProductVersion>
     <VisualStudioProductVersion Condition="'$(VisualStudioVersion)'=='14.0'">2015</VisualStudioProductVersion>
     <VisualStudioProductVersion Condition="'$(VisualStudioVersion)'=='15.0'">2017</VisualStudioProductVersion>
+	<VisualStudioProductVersion Condition="'$(VisualStudioVersion)'=='16.0'">2019</VisualStudioProductVersion>
```

Comment:
Unless I'm mistaken `<VisualStudioProductVersion/>` is only used in `CefSharp` so you can set this to `2017` and remove the changes to the `vcxproj` files. Also the indentation needs fixing.